src/JustSaying/AwsTools/QueueCreation/AmazonQueueCreator.cs

```diff
@@ -109,13 +108,27 @@ public QueueWithAsyncStartup<SqsQueueByName> EnsureQueueExists(string region, Sq
 
         private static async Task SubscribeQueueAndApplyFilterPolicyAsync(
             IAmazonSimpleNotificationService amazonSimpleNotificationService,
-            string topicArn, IAmazonSQS amazonSQS, Uri queueUrl, string filterPolicy)
+            string topicArn,
+            IAmazonSQS amazonSQS,
+            Uri queueUrl,
+            string filterPolicy)
         {
-            var subscriptionArn = await amazonSimpleNotificationService.SubscribeQueueAsync(topicArn, amazonSQS, queueUrl.AbsoluteUri)
+            if (amazonSimpleNotificationService == null)
```

Comment:
Don't necessarily have to change this, but I'm not a fan of the "line width" rule creating different indentation style for all these guard clauses.

Would be better either to have them all with braces, all indented below, or all on the same line. Having two different styles for 4 lines of code isn't great.

Reply:
Yeah I think you're right, the line limit has started annoying me a bit too. Will turf it 👍

Reply:
Have removed the line wrap rule, and fixed up these checks 👍